R/support_panel.R

```diff
@@ -14,9 +14,21 @@
 #' @param alt_href Alternative link to the parent publication (if not hosted on
 #' Explore Education Statistics)
 #' @param form_url URL for a feedback form for the dashboard
+#' @param feedback_custom_text A single vector or a combined vector wrapped in
+#' `shiny::tagList()` instead of c() for custom text to go under the "Give us
+#' feedback" heading.
+#' @param info_custom_text A single vector or a combined vector wrapped in
+#' `shiny::tagList()` instead of c() for custom text to go under the
+#' "Find out more information on the data" heading.
+#' @param contact_custom_text A single vector or a combined vector wrapped in
```

Comment:
I'm not sure we'd want to allow users to edit the contact bit too much, so I'd be tempted to drop this argument, don't think there was a specific ask for this in #54 and having a consistent approach to contact details is probably better for users